packages/kit/src/types/public.d.ts

```diff
@@ -18,8 +16,9 @@ import {
 	RequestOptions,
 	RouteSegment,
 	UniqueInterface
-} from './private.js';
+} from './private';
```

Comment:
AFAIK all type imports need the `.js` ending or else `moduleResolution: node16/nodenext` will error. It's stupid, but it is what it is.